website/alicloud.erb

```diff
@@ -1020,6 +1020,16 @@
                         </li>
                     </ul>
                 </li>
+
+               <li<%= sidebar_current("docs-alicloud-resource-yundun_dbaudit-instance") %>>
+                    <a href="#">Cloud DBaudit Resources</a>
+                    <ul class="nav">
+                        <li<%= sidebar_current("docs-alicloud-resource-yundun_dbaudit-instance") %>>
+                            <a href="/docs/providers/alicloud/r/yundun_dbaudit_instance.html">alicloud_dbaudit_instance</a>
+                        </li>
+                    </ul>
+                </li>
+
```

Comment:
please register bosh data source and resource's docs in the `alicloud.erb`

Reply:
will do